### DISCUSSION TITLE
Win32-OpenSSH ChrootDirectory and Shared Directories for file drop

### DISCUSSION THREAD

**Discussion by holzert** [PowerShell/Win32-OpenSSH#2169] [Q&A] [answered]
Running OpenSSH on Windows Server 2016.  Using Renci.SSHNet as sftp client.  Trying to use Renci SSHNet SFTP to transfer files to a directory share on the OpenSSH server.
ChrootDirectory /FileDrop

doesn't work.  
Nor does //FileDrop
Nor does / 
   and then using SFTP to change to /FileDrop or //FileDrop

How to use a mapped network directory e.g. c:\AllDroppedFiles shared as /FileDrop ?  OpenSSH service will not even start when ChrootDirectory in sshd_config file is set to /FileDrop or //FileDrop etc
Only paths I can get to work seem to be true paths on a drive e.g. c:\AllDroppedFiles will work.

**Comment by maertendMSFT** [comment, marked as answer]
This is because you are trying to access a file share, there are currently issues with accessing and authenticating to a file share over ssh.

https://github.com/PowerShell/Win32-OpenSSH/issues/1734
https://github.com/PowerShell/Win32-OpenSSH/issues/1561
